tripy/tripy/frontend/module/module.py

```diff
@@ -276,13 +276,13 @@ def _iterate_members_of_type(self, typ: T) -> Iterator[Tuple[str, T]]:
         for name, value in vars(self).items():
             if isinstance(value, typ):
                 yield name, value
-            elif isinstance(value, List) and _contains_types(value, [typ]) and _is_homogeneous_container(value):
+            elif isinstance(value, List) and _contains_types(value, [typ]) and _is_homogeneous_container(value, [Module, Parameter]):
```

Comment:
Shouldn't this be:
```suggestion
            elif isinstance(value, List) and _contains_types(value, [typ]) and _is_homogeneous_container(value, [typ]):
```
? 
Same below